docs/dev-setup/install.md

```diff
@@ -30,7 +30,8 @@ make peer
 
 To see what commands are available, simply execute the following commands:
 ```
-$ peer
+cd $GOPATH/src/github.com/hyperledger/fabric/build/bin
+./peer
 ```
```

Comment:
This document was written specifically for vagrant.  In that context, $GOPATH/src/github.com/hyperledger/fabric/build/bin is already in the users path (indirectly, via /hyperledger mountpoint) and therefore "peer" alone should be correct.